CMakeLists.txt

```diff
@@ -0,0 +1,381 @@
+# TODO: determine CMAKE_SYSTEM_NAME on OS/390.  Currently assumes "OS/390".
+cmake_minimum_required(VERSION 3.0)
+project(libuv)
```

Comment:
Maybe change the project name to be "uv" rather than "libuv". This is more in line with what the gyp build does.

Reply:
I surveyed what other cmake-based projects call their libraries and 'libfoo' - not 'foo' - seems to be the standard.